### DISCUSSION TITLE
Add underlines to links in comments, issues, discussions

### DISCUSSION THREAD

**Discussion by aardrian** [community/community#17284] [Accessibility] [answered]
Currently a link within a paragraph of text for an issue, comment, discussion, etc. relies on only a change in color to convey it is a link. This makes it difficult for some users to visually identify links without being forced to interact with them.

- In the dark mode version of the site, the text is light gray (`rgb(201, 209, 217)`) and the link text is blue (`rgb(88, 166, 255)`). These colors have a contrast ratio of 1.6:1 to one another.
- In the light mode version of the site, the text is dark gray (`rgb(36, 41, 47)`) and the link text is blue (`rgb(9, 105, 218)`). These colors have a contrast ratio of 2.8:1 to one another.

These are both below the 3:1 contrast ratio identified in WCAG 2.1 Success Criterion [1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html).

As such they either need to have a color change to meet the 3:1 contrast ratio, or some other visual cue. Underlines tend to perform best and are most recognizable (ref: _[On Link Underlines](https://adrianroselli.com/2016/06/on-link-underlines.html#G183)_) compared to other visual cues.

**Comment by benjiallen** [comment]
@aardrian thanks for this feedback. This is something we continue to work on. I'll create the internal issues and work with our design partners to see what we can do.

**Reply by aardrian** [reply]
Hey there, have your design partners come up with anything over the last year and three months?

**Comment by simevidas** [comment]
Wikipedia has an “always underline links” option (under Settings > Appearance). Could GitHub get such an option?

<img width="497" alt="Underline links: Skin or browser default, Never, Always." src="https://github.com/community/community/assets/716405/3f420961-7619-4da0-ac93-8c64ee082394">

**Comment by dipree** [comment, marked as answer]
Hi, we have a new [accessibility setting](https://github.com/settings/accessibility) that allows you to show underlines for links in text blocks. If you have any feedback, please let us know in this [discussion](https://github.com/orgs/community/discussions/68734). Thank you.

**Reply by yatil** [reply]
FINALLY.

**Comment by cehfisher** [comment]
Hi @aardrian - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/17284#discussioncomment-7142836). Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)
